server/recruitment.py

```diff
@@ -9,10 +10,25 @@ def get_questions():
 
 
 def get_answers(user_id):
-    return {
-        answer.question_id: answer.text
-        for answer in Answer.query(user_id=user_id).run()
-    }
+    questions = get_questions()
+    answers = {}
+    answer_query = Answer.query().where(Answer.user_id == user_id)
+    answer_list = answer_query.run()
+    for answer in answer_list:
+        answers[answer.question_id] = {
+            "question": questions[answer.question_id],
+            "user_id": user_id,
+            "answer": answer.text,
+        }
+    if not answers:
```

Comment:
As we discussed, this should be updated to add empty questions when the user has answered only a subset of all questions.

Reply:
added